mavlink-bindgen/src/parser.rs

```diff
@@ -1026,7 +1025,8 @@ impl MavType {
                 let w = Char.rust_writer(&quote!(*val), buf);
                 quote! {
                     let slice = #val.as_bytes();
-                    for val in slice {
+                    let remaining_capacity = #val.remaining_capacity();
+                    for val in slice.iter().chain(std::iter::repeat(&0u8).take(remaining_capacity)) {
```

Comment:
We can't use `std::iter::repeat` here, as we must be compliant with no-std builds.

Reply:
Indeed, I noticed the CI failure yesterday but didn't have the opportunity to fix until now. I changed it to use core, it should be good now.